packages/libs/client/vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    include: ['src/**/*.test.ts'],
+    exclude: ['src/**/*.int.test.ts'],
+    globals: true,
+    threads: false, // To prevent error in tests using jsdom environment: Module did not self-register: canvas.node
```

Comment:
I think this client package is not using the jsdom environment

Reply:
It's indeed not obvious, but some tests use it by this comment at the top of the file: `/** @vitest-environment jsdom */`